headless/src/jsMain/kotlin/dev/fritz2/headless/components/disclosure.kt

```diff
@@ -118,11 +116,8 @@ class Disclosure<C : HTMLElement>(tag: Tag<C>, id: String?) : Tag<C> by tag, Ope
         initialize: DisclosurePanel<CP>.() -> Unit
     ) {
         addComponentStructureInfo("disclosurePanel", this@disclosurePanel.scope, this)
-        tag(this, classes, "$componentId-panel", scope) {
-            DisclosurePanel(this).run {
-                initialize()
-                render()
-            }
+        panel = tag(this, classes, "$componentId-panel", scope) {
+            DisclosurePanel(this).run(initialize)
```

Comment:
`render` is no longer needed?

Reply:
That's correct. In this case the `render` function only contained code to dynamically set an attribute which has been moved elsewhere, leaving the function empty. Thats why I removed it altogether.

Reply:
Now I have seen it :-)